rpm/qtdeclarative.spec

```diff
@@ -338,6 +363,9 @@ cp lib/libQt5QmlDevTools.a %{buildroot}/%{_libdir}
 %{_datadir}/qt5/mkspecs/modules/qt_lib_qml_private.pri
 %{_libdir}/cmake/
 
+%files doc
+%{_docdir}/qt5/qtqml/
+%{_docdir}/qt5/qtqml.qch
```

Comment:
won't this fail if docs aren't being built? (since the file won't then exist)

I'd suggest maybe a
'touch %{buildroot}/%{_docdir}/qt5/qtqml', and change the %files list here to a wildcard on that dir (which won't fail if it's empty)